evaluate.cpp

```
@@ -548,6 +583,127 @@ BOOL Set_Up_Set_Strings (const int set_type,
+      liveMap.SetAt (changes [iApplied].strName, changes [iApplied].pNew);
+
+    // Build all runtime indexes while removed objects are still recoverable.
+    (pDoc->*sortObjects) (&excludedObjects);
+    }
+  catch (...)
+    {
+    while (iApplied > 0)
+      {
+      iApplied--;
+      CSetPublishChange<TObject> & change = changes [iApplied];
+      if (change.pOld)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)